modules/tester/tester.go

```diff
@@ -5,14 +5,14 @@ import (
 	"path/filepath"
 )
 
-const (
-	SiaTestingDir = "SiaTesting"
+var (
+	SiaTestingDir = filepath.Join(os.TempDir(), "SiaTesting")
 )
 
-// TempDir takes a set of directory names and joins them to the sia temp
-// folder.
+// TempDir joins the provided directories and prefixes them with the Sia
+// testing directory.
 func TempDir(dirs ...string) string {
-	tmp := append([]string{os.TempDir()}, SiaTestingDir)
-	full := append(tmp, dirs...)
-	return filepath.Join(full...) // filepath.Join(tmp, testing, dirs...) returns 'too many arguments' error.
+	path := filepath.Join(SiaTestingDir, filepath.Join(dirs...))
+	os.RemoveAll(path) // remove old test data
```

Comment:
The problem with doing it in the Makefile is that it's not cross-platform. See the comment on c8b2e441f9cb6aa52909caf7aa1fcaa17a6cb061